pkg/webhooks/validation/controller/controller.go

```diff
@@ -364,6 +364,10 @@ func (c *Controller) reconcileRequest(req *reconcileRequest) error {
 }
 
 func (c *Controller) readyForFailClose() bool {
+	//for remote clusters, we will set fail-close by default
```

Comment:
It would be better to explain why we do this rather than what we are doing

Reply:
fixed, PTAL, thanks.